test/test_environment.lua

```diff
@@ -572,8 +661,8 @@ function test_env.main()
 
    local install_env_vars = {
       LUAROCKS_CONFIG = test_env.testing_paths.testing_dir .. "/testing_config.lua",
-      LUA_PATH = "",
-      LUA_CPATH = ""
+      LUA_PATH,
```

Comment:
The two lines after the change do not do anything, `{LUAROCKS_CONFIG = ..., LUA_PATH, LUA_CPATH}` == `{LUAROCKS_CONFIG = ...}` because `LUA_PATH` and `LUA_CPATH` are undefined globals